autoPyTorch/pipeline/components/training/trainer/StandardTrainer.py

```diff
@@ -23,39 +28,42 @@ def __init__(self, weighted_loss: bool = False,
         super().__init__(random_state=random_state)
         self.weighted_loss = weighted_loss
 
-    def data_preparation(self, X: np.ndarray, y: np.ndarray,
-                         ) -> typing.Tuple[np.ndarray, typing.Dict[str, np.ndarray]]:
+    def _data_preprocessing(self, X: torch.Tensor, y: torch.Tensor,
+                            ) -> Tuple[torch.Tensor, _NewLossParameters]:
         """
         Depending on the trainer choice, data fed to the network might be pre-processed
         on a different way. That is, in standard training we provide the data to the
         network as we receive it to the loader. Some regularization techniques, like mixup
         alter the data.
 
         Args:
-            X (np.ndarray): The batch training features
-            y (np.ndarray): The batch training labels
+            X (torch.Tensor): The batch training features
+            y (torch.Tensor): The batch training labels
 
         Returns:
-            np.ndarray: that processes data
-            typing.Dict[str, np.ndarray]: arguments to the criterion function
+            torch.Tensor: that processes data
+            _NewLossParameters: arguments to the new loss function
         """
-        return X, {'y_a': y}
+        return X, _NewLossParameters(y_a=y)
 
-    def criterion_preparation(self, y_a: np.ndarray, y_b: np.ndarray = None, lam: float = 1.0
-                              ) -> typing.Callable:
+    def _get_new_loss_fn(
+        self,
+        new_loss_params: _NewLossParameters
+    ) -> Callable:
+        y_a = new_loss_params.y_a
         return lambda criterion, pred: criterion(pred, y_a)
 
     @staticmethod
-    def get_properties(dataset_properties: typing.Optional[typing.Dict[str, typing.Any]] = None
-                       ) -> typing.Dict[str, str]:
+    def get_properties(dataset_properties: Optional[Dict[str, Any]] = None
+                       ) -> Dict[str, str]:
         return {
             'shortname': 'StandardTrainer',
             'name': 'StandardTrainer',
```

Comment:
```suggestion
            'name': 'Standard Trainer',
```

Reply:
Why do we want space here?

Reply:
its not important, but in general we have followed that short name is either acronym of the whole name or the whole name without spaces. You can definitely ignore this but I suggested for uniformity

Reply:
Ok, I do not know where we are using those names, but if it is convention, I will follow this rule.